src/main/java/it/gov/pagopa/pu/processecexutions/mapper/ExportFileRequestMapper.java

```diff
@@ -0,0 +1,21 @@
+package it.gov.pagopa.pu.processecexutions.mapper;
+
+import it.gov.pagopa.pu.processecexutions.dto.generated.ExportFileRequestDTO;
+import it.gov.pagopa.pu.processecexutions.enums.ExportFileStatus;
+import it.gov.pagopa.pu.processecexutions.model.ExportFile;
+import org.springframework.stereotype.Service;
+
+@Service
+public class ExportFileRequestMapper {
+
+  public <T extends ExportFile<?>> T map(ExportFileRequestDTO dto, String operatorExternalId, T exportFile) {
+      exportFile.setOrganizationId(dto.getOrganizationId());
+      exportFile.setOperatorExternalId(operatorExternalId);
+      exportFile.setFilePathName(dto.getFilePathName());
+      exportFile.setFileName(dto.getFileName());
+      exportFile.setFileSize(dto.getFileSize());
+      exportFile.setStatus(ExportFileStatus.REQUESTED); // TODO: correct status?
```

Comment:
Yes, as you can see here:
https://pagopa.atlassian.net/wiki/spaces/SPAC/pages/1470169164/Flussi+di+caricamento+e+download+dei+file#3.-Export-dei-dati
```suggestion
      exportFile.setStatus(ExportFileStatus.REQUESTED);
```